README.md

```diff
@@ -47,8 +54,7 @@ Execute the commands below to build from source.
 
 As an open source project, Ballerina welcomes contributions from the community. 
 
-You can also check for [open issues](https://github.com/ballerina-platform/module-ballerina-config/issues) that interest
- you. We look forward to receiving your contributions.
+You can also check for [open issues](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fconfig) that interest you. We look forward to receiving your contributions.
```

Comment:
According to ballerina-platform/module-ballerina-http#99 this line has to be removed.

Reply:
Fixed in 0b12510